src/kakarot/execution_context.cairo

```diff
@@ -35,6 +36,29 @@ namespace ExecutionContext {
         evm_contract_address: felt,
     }
 
+    // @notice Initialize an empty context to act as a placeholder for root context
+    // @return An stopped execution context
+    func init_root() -> model.ExecutionContext* {
+        let (root_context) = get_label_location(empty_context);
+        let ctx = cast(root_context, model.ExecutionContext*);
+        return ctx;
+
+        empty_context:
```

Comment:
are we adopting this low-level notation to optimise or by necessity?

Reply:
it's not really a low level notation, it's a (the) way to define a constant array in cairo. The other option would be

```
empty_context = alloc()
assert empty_context[0] = 0
assert empty_context[1] = 0
assert empty_context[2] = 0
...
```

I like better the `dw` because it's more concise and explicit: I'm defining a constant

Reply:
sure:)